install/linux/usr/share/odemis/sim/fastem-sim.yaml

```diff
@@ -0,0 +1,64 @@
+# Minimal configuration file to start the Fast-EM GUI
+FASTEM-sim: {
+    class: Microscope,
+    role: mbsem,
+    children: ["SEM", "EBeam Scanner", "EBeam Focus", "SED",
+               "Sample Stage"],
+}
+
+"SEM": {
+    class: simsem.SimSEM,
+    # TODO remove role when children components are implemented (chamber camera, chamber and sed)
+    role: sem,
+    init: {
+           image: "simsem-fake-output.h5", # any large 16 bit image is fine
+           drift_period: 5, # sec
```

Comment:
If you remove this argument, the image will be larger, and not drift, which is probably closer to the reality.